Gemfile

```diff
@@ -42,4 +42,5 @@ group :test do
   gem 'rails-controller-testing'
   gem 'simplecov', '~> 0.16.1'
   gem 'webmock', '~> 3.4.2'
+  gem 'dalli', '~> 2.7.9'
```

Comment:
This is under group test as it is only used for testing locally. Dalli is used for cacheing during tests